<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordCracker</class>
 <widget class="QWidget" name="PasswordCracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password Cracker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_password">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="passwordInput"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_salt">
       <property name="text">
        <string>Salt (2 characters):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="saltInput"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="hashLabel">
     <property name="text">
      <string>Generated Hash:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="threadStatus">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_start">
     <property name="text">
      <string>Start Cracking</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

